services/jupyter-server/Dockerfile

```
@@ -5,43 +5,26 @@ RUN apt-get update && apt-get install -y ca-certificates && update-ca-certificat
 # install nodejs for jupyterextension install support
 RUN curl -sL https://deb.nodesource.com/setup_14.x | bash - && apt-get install -y nodejs
 
-RUN pip3 install git+https://github.com/jupyter/notebook.git@b44fc3b01dd1ae059170ab9ea657e34d44f5e36c
-RUN pip3 install flask_socketio \
+RUN pip3 install pip --upgrade
+
+RUN pip3 install jupyterlab==3.0.0rc9 \
+    flask_socketio \
```

```suggestion
    flask_socketio \
    requests \
```
Should we pin down the versions?